.gitignore

```diff
@@ -6,3 +6,4 @@ Cargo.lock
 *.csv
 *.json
 *.jsonl
+.DS_Store
```

Comment:
@diana-qing I think this needs to be *.DS_Store so that it also catches files in subdirectories